backend/vxlan/utils.go

```diff
@@ -151,3 +151,50 @@ func getHostsMap(hosts []metadata.Host) map[string]metadata.Host {
 	logrus.Debugf("hostsMap: %v", hostsMap)
 	return hostsMap
 }
+
+func getLocalRouterInfo(networks []metadata.Network, host metadata.Host, services []metadata.Service) (metadata.Network, metadata.Container, metadata.Service) {
+	var (
+		vxlanNetwork metadata.Network
+		localRouter  metadata.Container
+		selfService  metadata.Service
+	)
+	for _, aNetwork := range networks {
```

Comment:
In future we want to support multiple network drivers in the same environment. I guess this logic needs change to accommodate that.